content/tutorials/manage-the-rippled-server/configuration/configure-history-sharding.md

```diff
@@ -6,36 +6,41 @@
 
 To configure your `rippled` to store shards of ledger history, complete the following steps:
 
-## 1. Determine how much space to allocate for the shard store
+## 1. Determine how much many shards to maintain
 
-Before you configure your `rippled` server to store history shards, you must decide how much disk space to allocate to the history shard store. This also affects how much history you keep in the default ledger store. You should consider the following when deciding what size to configure your shard store:
+Before you configure your `rippled` server to store history shards, you must decide how many history shards you want to keep, which is mostly determined by how much disk space have available for use in the shard store. This also affects how much history you keep in the default ledger store. You should consider the following when deciding what size to configure your shard store:
```

Comment:
"disk space have available for use in the" -> "disk space is available for use in the"